### DISCUSSION TITLE
Content Editor Icons - Columns

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2888] [Request: Icon] [answered]
Hi! I need to figure out column icons for the content editor, and wanted to confirm that this is the right place to start. Streamline doesn't have all of the icons needed, but I'm wondering if I started with something like this Streamline Light icon:

![Screenshot 2022-12-14 at 9 53 21 AM](https://user-images.githubusercontent.com/87141551/207657992-ff13d277-111c-44de-a3a8-64b6a161ac80.png)

I could build out the rest:

1. 1 Column
2. 2 Columns
3. 3 Columns
4. 4 Columns
5. 1:2 Columns
6. 2:1 Columns
7. 1:3 Columns
8. 3:1 Columns

Current SG Column icons:

![Screenshot 2022-12-14 at 9 53 54 AM](https://user-images.githubusercontent.com/87141551/207658119-57cb65df-ac3f-471e-9b8b-1d58ae5d6f73.png)

Please let me know if there's a better Streamline set to build from, thanks!

**Comment by nkrantz** [comment, marked as answer]
Hey @ceyerkes! Thanks for sharing the update.

[Streamline's "Core Line - Free" set](https://www.streamlinehq.com/icons/streamline-mini-line?search=column) has quite a few layout icons that you could use for inspiration. Right now, there isn't much of a need for these to live in Paste since they're pretty product-specific, so we'd recommend moving forward with the icons on your end.

For more info on how to export from Figma and svg-to-react package, you can use the guidance on our docs site ([contribution guide](https://paste.twilio.design/introduction/contributing/icons#setting-up-an-icon-in-figma), [svg-to-react](https://github.com/twilio-labs/svg-to-react)).

Please keep us updated!

**Reply by ceyerkes** [reply]
Will do, thanks @nkrantz !
